src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -83,9 +82,6 @@ export default class PatientViewMutationTable extends MutationTable<IPatientView
             sortBy:(d:Mutation[])=>TumorColumnFormatter.getSortValue(d, this.props.sampleManager)
         };
 
-        // patient view has a custom renderer for protein change column, so we need to override render function
```

Comment:
we don't need to override this anymore, now we use the same formatter for both tables